front-end/src/renderer/pages/Settings/components/KeysTab.vue

```diff
@@ -47,15 +48,20 @@ enum Tabs {
 /* State */
 const isDeleteModalShown = ref(false);
 const isUpdateNicknameModalShown = ref(false);
+const deleteAll = ref(false);
+const isSelectAll = ref(false);
+const selectedKeyPairIdsToDelete = ref<string[]>([]);
+const selectedMissingKeyPairIdsToDelete = ref<string[]>([]);
```

Comment:
The `selectedMissingKeyPairIdsToDelete` is a string array.
The missing key pair ids are ids from the organization, which are ints.
You can store them directly in type number and avoid parsing the to Number in `handleDelete`.